arches/app/media/js/views/components/widgets/radio-boolean.js

```diff
@@ -27,14 +28,24 @@ define([
          
         WidgetViewModel.apply(this, [params]);
         var self = this;
+        this.ariaCheckedTrueValue = ko.observable();
```

Comment:
we should be able to remove these variables and all the associated code, as we can just compare the value against `true` or `false` in the UI